NitroxPatcher/Patches/Persistent/PAXTerrainController_LoadAsync_Patch.cs

```diff
@@ -34,34 +64,46 @@ public static IEnumerable<CodeInstruction> Transpiler(MethodBase original, IEnum
 
         public override void Patch(HarmonyInstance harmony)
         {
-            PatchTranspiler(harmony, getMethod());
+            PatchTranspiler(harmony, GetMethod());
+        }
+
+        private static FieldInfo GetStateField(IReflect type)
```

Comment:
I believe there is a GetField in reflection helper

Reply:
That GetField in reflection helper unfortunately wont help with Async methods